frontend/src/components/widgets/BaseWidget/styled-components.ts

```diff
@@ -12,6 +12,12 @@ export const StyledWidgetLabelHelp = styled.div(() => ({
   right: 0,
 }))
 
+export const StyledWidgetLabelHelpTopLeft = styled.div(() => ({
+  position: "absolute",
+  left: "-30px",
```

Comment:
One issue with positioning them on the top left like this is that the tooltips get cut off on mobile:

![image](https://user-images.githubusercontent.com/856034/105246690-9bcf7b80-5b41-11eb-8fa6-64a0d554a8b4.png)